.ci/daily-config.yaml

```diff
@@ -4,25 +4,99 @@ dependencies: ./project-dependencies.yaml
 
 pre: |
   export MAVEN_COMMAND=mvn -B -e -U clean deploy -Dfull -Drelease -DaltDeploymentRepository=local::default::file://${{ env.WORKSPACE }}/deploy-dir -Dmaven.test.redirectTestOutputToFile=true -Dmaven.test.failure.ignore=true -Dgwt.memory.settings="-Xmx10g" -Prun-code-coverage -Dcontainer.profile=wildfly -Dcontainer=wildfly -Dintegration-tests=true -Dmaven.wagon.httpconnectionManager.ttlSeconds=25 -Dmaven.wagon.http.retryHandler.count=3
-  echo "MAVEN_COMMAND=" ${{ env.MAVEN_COMMAND }}
-  echo "WORKSPACE=" ${{ env.WORKSPACE }}
-  java -version
 
 default:
   build-command:
     current: |
+      ${{ env.WORKSPACE }}/bc/kiegroup_droolsjbpm-build-bootstrap/script/release/update-versions-build-chain.sh ${{ env.KIE_VERSION }} ${{ env.REPOSITORY }}
       ${{ env.MAVEN_COMMAND }}
       ${{ env.WORKSPACE }}/clean-up.sh `pwd`
-
 build:
   - project: kiegroup/lienzo-core
     build-command:
       before:
-        current: |
-          ${{ env.WORKSPACE }}/bc/kiegroup_droolsjbpm-build-bootstrap/script/release/update-versions-build-chain.sh ${{ env.KIE_VERSION }} community
-      current: |
-        ${{ env.MAVEN_COMMAND }}
-        ${{ env.WORKSPACE }}/clean-up.sh `pwd`
+        current:
+          export REPOSITORY=lienzo-core
+  - project: kiegroup/lienzo-tests
+    build-command:
+      before:
+        current:
+          export REPOSITORY=lienzo-tests
```

Comment:
Instead of having to export the repository for each projects, have you checked if there is a way to get it from the git folder?
I mean all projects specific commands are executed inside the git folder repo of that project so maybe something like `basename -s .git $(git config --get remote.origin.url)` might work and can be generalized in the default command (did not try yet).